views/ministryteams.ejs

```diff
@@ -0,0 +1,103 @@
+<!doctype html>
+<html lang="en">
+  
+<%- include widgets/header.ejs %>
+<%- include widgets/mainNav.ejs %>
+<div class="overlay" id="our-beliefs">
+  <h1 id="top">Ministry Teams</h1>
+</div>
+
+
+<div class="container">
+  <div class="row">
+      <div class="col-md-8 mx-auto px-4 py-5 my-3 order-md-1">
+        <h3>List of Ministry Teams</h3>
+        <hr>
+        <h5 id = 0>
+        &nbsp&#9830
+        <% teams.forEach(function(Team, index){%>
+          <a href=#<%=index%>><%=Team.name%></a>&nbsp&#9830&nbsp
```

Comment:
We could just include the individual team links as part of a `<ul>` block with `<li><a>...</a></li>`. This will make it more ordered and consistent stylistically; also idk if other browsers support this special character. It's up to you though.